CHANGES.rst

```diff
@@ -1,8 +1,11 @@
 Changelog
 =========
+..
+    To be released
+    --------------
```

Comment:
Looks like the indentation is broken?

Reply:
Commented the `To be released` heading because it looks sloppy when there is a new release and the heading is empty.